src/makeCalculator.js

```diff
@@ -37,7 +37,46 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add: function(value) {
+      result += value;
+
+      return this;
+    },
+    subtract: function(value) {
+      result -= value;
+
+      return this;
+    },
+    multiply: function(value) {
+      result *= value;
+
+      return this;
+    },
+    divide: function(value) {
```

Comment:
```suggestion
    multiply: function(value) {
      result *= value;

      return this;
    },
    
    divide: function(value) {
```

Add blank line between methods